plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml

```diff
@@ -220,7 +220,7 @@ Item
         }
         border
         {
-            color: printer && printer.activePrintJob && printer.activePrintJob.configurationChanges.length > 0 ? "#f5a623" : "transparent" // TODO: Theme!
+            color: printer && printer.activePrintJob && printer.activePrintJob.configurationChanges.length > 0 ? UM.Theme.getColor("warning") : "transparent" // TODO: Theme!
             width: borderSize // TODO: Remove once themed
         }
         color: "transparent" // TODO: Theme!
```

Comment:
Should this be put into the theme?

Reply:
Transparent not, as it just needs to not be visible sometimes and it's cleaner than to make two separate elements for it.